target/linux/ramips/image/mt76x8.mk

```diff
@@ -319,6 +320,17 @@ define Device/tplink_archer-c50-v4
 endef
 TARGET_DEVICES += tplink_archer-c50-v4
 
+define Device/tplink_re200-v2
+  $(Device/tplink-safeloader)
+  IMAGE_SIZE := 6016k
```

Comment:
7808k?

Reply:
Yep, you're right. Copied that from the RE305-v1 without paying attention on the partition sizes.